BUILD.md

```diff
@@ -235,40 +254,7 @@ conan profile update env.CC=<path> default
 conan profile update env.CXX=<path> default
 ```
 
-
-## How to build and test
-
-Let's start with a couple of examples of common workflows.
-The first is for a single-configuration generator (e.g. Unix Makefiles) on
-Linux or MacOS:
-
-```
-conan export external/rocksdb
-mkdir .build
-cd .build
-conan install .. --output-folder . --build missing --settings build_type=Release
-cmake -DCMAKE_TOOLCHAIN_FILE:FILEPATH=build/generators/conan_toolchain.cmake -DCMAKE_BUILD_TYPE=Release ..
-cmake --build .
-./rippled --unittest
-```
-
-The second is for a multi-configuration generator (e.g. Visual Studio) on
-Windows:
-
-```
-conan export external/rocksdb
-mkdir .build
-cd .build
-conan install .. --output-folder . --build missing --settings build_type=Release --settings compiler.runtime=MT
-conan install .. --output-folder . --build missing --settings build_type=Debug --settings compiler.runtime=MTd
-cmake -DCMAKE_TOOLCHAIN_FILE:FILEPATH=build/generators/conan_toolchain.cmake ..
-cmake --build . --config Release
-cmake --build . --config Debug
-./Release/rippled --unittest
-./Debug/rippled --unittest
-```
```

Comment:
Why are these instructions removed? To what steps do the numbered explanations correspond?

Reply:
@justinr1234 has probably removed them by mistake. I put them back.

Reply:
It was removed when there wasn’t a separate readme.

As far as these instructions, it shouldn’t have Debug and Release together like that as both myself and @drlongle ran into issues with mixing debug and release builds back to back.